scripts/setup-ts-execution.js

```diff
@@ -3,8 +3,13 @@ const babel = require(`@babel/core`);
 const os = require(`os`);
 const root = path.dirname(__dirname);
 
+// The cache in @babel/register never clears itself and will therefor grow
```

Comment:
I think `therefore` would be the correct word here.

```suggestion
// The cache in @babel/register never clears itself and will therefore grow
```

Reply:
It is indeed 👍